builds/versions-win64.txt

```diff
@@ -0,0 +1,19 @@
+mono-version =

```

Comment:
Why is this now populating?

Reply:
Gah didn't mean to commit this file I'll remove it. Nice catch.